Sources/Plasma/FeatureLib/pfDXPipeline/plDXDevice.cpp

```diff
@@ -161,6 +183,257 @@ void plDXDevice::SetViewport()
     WEAK_ERROR_CHECK(fD3DDevice->SetViewport(&vp));
 }
 
+/**
+ * Calculate the vertex stride from the given format.
+ */
+static uint32_t GetBufferFormatSize(uint8_t format)
+{
+    uint32_t  size = sizeof(float) * 6 + sizeof(uint32_t) * 2; // Position and normal, and two packed colors
```

Comment:
```suggestion
    constexpr uint32_t size = sizeof(float) * 6 + sizeof(uint32_t) * 2; // Position and normal, and two packed colors
```

Reply:
This can't be `constexpr` because we add to the `size` variable and then return it